package.json

```diff
@@ -3,6 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "devDependencies": {
+    "enzyme": "^2.8.0",
```

Comment:
@DavidKnott Enzyme is our way to do Capybara/browser type stuff. You can `shallow`, `mount`, and `render` a component and run assertions on that state. The [docs](http://airbnb.io/enzyme/docs/api/) are super helpful.